packages/ardrive_utils/lib/src/html/implementations/html_web.dart

```diff
@@ -26,10 +26,16 @@ StreamSubscription<Event> onTabGetsFocused(Function onFocus) {
   return subscription;
 }
 
-void onWalletSwitch(Function onWalletSwitch) {
+Function() onWalletSwitch(Function onWalletSwitch) {
   window.addEventListener('walletSwitch', (event) {
     onWalletSwitch();
   });
+
+  return () {
+    window.removeEventListener('walletSwitch', (event) {
+      onWalletSwitch();
+    });
+  };
 }
```

Comment:
IMO it's still wrong - we're now passing two different instances of anonymous functions.
```suggestion
Function() onWalletSwitch(Function onWalletSwitch) {
  function listener(event) {
    onWalletSwitch();
  };

  window.addEventListener('walletSwitch', listener);
  return () => window.removeEventListener('walletSwitch', listener);
}
```

Reply:
7027f1b3f3f1e0c7ff40d3c511e8ddf987d6c6c0